eq-author/src/components/ContentPickerv2/VariablePicker.js

```diff
@@ -0,0 +1,140 @@
+import React from "react";
+import PropTypes from "prop-types";
+import CustomPropTypes from "custom-prop-types";
+
+import styled from "styled-components";
+
+import { colors } from "constants/theme";
+import {
+  SubMenuItem,
+  MenuItemType,
+  MenuItemTitle,
+  MenuItemSubtitle,
+} from "./Menu";
+import Truncated from "components/Truncated";
+
+import ScrollPane from "components/ScrollPane";
+
+const ModalTitle = styled.div`
+  font-weight: bold;
+  font-size: 1.2em;
+  color: ${colors.textLight};
+`;
+
+const ModalHeader = styled.div`
+  padding: 2em 1em;
+  border-bottom: 1px solid ${colors.bordersLight};
+`;
+
+const MenuContainer = styled.div`
+  overflow: hidden;
+  height: 25em;
+`;
+
+const Table = styled.div`
+  display: table;
+  width: 100%;
+`;
+
+const RightPositioner = styled.div`
+  float: right;
+`;
+
+const TableHeader = styled.div`
+  display: table-header-group;
+  background: #f4f5f6;
+  font-size: 11px;
```

Comment:
Do you think this would be better as `em` instead of `px`?

Reply:
That was the size in the design and 11px is a little neater than 0.6875em.